tests/tasks/tkn-bundle.go

```diff
@@ -0,0 +1,395 @@
+package tasks
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/google/go-containerregistry/pkg/authn"
+	"github.com/google/go-containerregistry/pkg/crane"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	kubeapi "github.com/redhat-appstudio/e2e-tests/pkg/clients/kubernetes"
+	"github.com/redhat-appstudio/e2e-tests/pkg/constants"
+	"github.com/redhat-appstudio/e2e-tests/pkg/framework"
+	"github.com/redhat-appstudio/e2e-tests/pkg/utils"
+	"github.com/tektoncd/cli/pkg/bundle"
+	pipeline "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+var _ = framework.TaskSuiteDescribe("tkn bundle task", Label("tasks", "HACBS"), func() {
```

Comment:
AC in EC-456 mention that this test should run as a part of build-definitions CI. So I guess there's a plan to update the [e2e taskrun](https://github.com/redhat-appstudio/build-definitions/blob/ffc53980a69641def06e90e188bceeb754b2d55d/.tekton/tasks/e2e-test.yaml#L29) to run tests with "tasks" label too?

Reply:
That or I can add the `build-templates-e2e` label to this. I just need to figure out how to test the task in the pull request. See https://github.com/redhat-appstudio/e2e-tests/pull/1127#discussion_r1581254534 for the questions I'm unsure about.

Reply:
since the linked comment is resolved, does it make sense to add the `build-templates-e2e` label here?

Reply:
It does and I'm thinking it makes sense to move this to `tests/build`. I'm doing that now and testing.